gradle.properties

```diff
@@ -65,8 +65,8 @@ reflectionsVersion=0.9.10
 guavaVersion=19.0
 groovyVersion=2.4.6
 
-springWebmvcPac4jVersion=1.0.2
-pac4jVersion=1.8.8
+springWebmvcPac4jVersion=1.1.0
```

Comment:
You did the upgrade to pac4j v1.9 in the same PR: GREAT!

You may want to wait for the version 1.1.1 of `springwebmvcpac4j` and version 1.9.1 of `pac4j` (coming very soon) as I will slightly change the signature of the `Authenticator` and `ProfileCreator` components.